Packs/VMRay/README.md

```diff
@@ -1 +1,15 @@
-*Note: Support for this pack will be moving to the partner around Oct 15, 2021.*
```

Comment:
Add 
*Note: Support for this pack moved to the Partner on December, 21, 2021. Please contact the Partner directly via the support link on the right.*